src/main/java/io/cdap/e2e/pages/locators/CdfStudioLocators.java

```diff
@@ -74,4 +74,26 @@
 
   @FindBy(how = How.XPATH, using = "//*[@data-cy='preview-configure-run-btn']")
   public static WebElement configRun;
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='valium-banner-hydrator']")
+  public static WebElement pipelineSaveSuccessBanner;
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='plugin-BigQueryTable-batchsource']")
+  public static WebElement sourceBigQuery;
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='plugin-GCS-batchsink']")
+  public static WebElement gcs;
+
+  @FindBy(how = How.XPATH, using = "//*[contains(@class,'plugin-endpoint_BigQuery')]")
+  public static WebElement fromBigQuery;
+
+  @FindBy(how = How.XPATH, using = "//*[@title='GCS']")
+  public static WebElement toGCS;
+
+  @FindBy(how = How.XPATH, using = "//*[@title=\"GCS\"]//following-sibling::div")
+  public static WebElement gcsProperties;
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='valium-banner-hydrator']")
+  public static WebElement previewSuccessBanner;
+
 }
```

Comment:
file doesnt end with new line.
verify checkstyle file has been included in intellij. 
Checkstyle settings can be downloaded from https://cdap.atlassian.net/wiki/spaces/CE/pages/1594098619/Coding+Standard